tests/conftest.py

```diff
@@ -632,3 +633,30 @@ def shuffle(request):
 def configure_shuffling(shuffle):
     with dask.config.set(shuffle=shuffle):
         yield
+
+
+@pytest.fixture
+def read_parquet_with_pyarrow():
+    """Force dask.dataframe.read_parquet() to return strings with dtype=string[pyarrow]
+    FIXME https://github.com/dask/dask/issues/9840
+    """
```

Comment:
CC @jrbourbeau at the moment this hacky fixture is used exclusively by the xgboost preprocess test.